src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Logging best practices guide
+tags:
+ - New Relic solutions
+ - Best practices guides
+ - Logs
+ - Logging
+translate:
+ - jp
+metaDescription: 'Best practices for using New Relic logs'
+redirects:
+---
+
+import parsingExample from 'images/logs-parsing-example.png'
```

Comment:
@marcelschlapfer @lchapman4 Can you point me to where this image is? I didn't see it in the Google doc.